test/calibration/test_calibrations.py

```diff
@@ -1612,3 +1612,51 @@ def test_alternate_initialization(self):
         )
 
         self.assertEqual(str(cals1.get_schedule("x", 1)), str(cals2.get_schedule("x", 1)))
+
+
+class TestSerialization(QiskitExperimentsTestCase):
+    """Test the serialization of the Calibrations."""
+
+    def test_serialization(self):
+        """Test the serialization."""
+
+        backend = FakeBelem()
+        library = FixedFrequencyTransmon(basis_gates=["sx", "x"])
+
+        cals = Calibrations.from_backend(backend, library)
+        cals.add_parameter_value(0.12345, "amp", 3, "x")
+
+        self.assertRoundTripSerializable(cals, self.json_equiv)
+
+    def test_equality(self):
+        """Test the equal method on calibrations."""
+        backend = FakeBelem()
+        library = FixedFrequencyTransmon(basis_gates=["sx", "x"])
+
+        cals1 = Calibrations.from_backend(backend, library)
+
+        date_time = datetime.now(timezone.utc).astimezone()
+        param_val = ParameterValue(0.12345, date_time=date_time)
+        cals1.add_parameter_value(param_val, "amp", 3, "x")
+
+        # The two objects are different due to missing parameter value
+        cals2 = Calibrations.from_backend(backend, library)
+        self.assertFalse(cals1 == cals2)
+
+        # The two objects are different due to time stamps
+        cals2.add_parameter_value(0.12345, "amp", 3, "x")
```

Comment:
See here: [2e78a86](https://github.com/Qiskit/qiskit-experiments/pull/656/commits/2e78a86d0a69c686131a131193a107d9b82a6b11) I also realized that `add_parameter_defaults=False` was needed since otherwise the tests were always passing irrespective of the parameters because the data time of the added default values were different.